### ISSUE TITLE
Sign up  New issue  [Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Screen Reader: Jaws not reading the dialogue name 'Create new scala class'

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3623]
[Check out Accessibility Insights!](https://github.com/microsoft/azure-tools-for-java/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS1.3.1

**Repro Steps:**
1)Turn on the screen reader (JAWS)
2)Launch IntelliJ IDEA 
3)Tab till Create New Project button and press enter
4)Tab till Azure Spark/HDInsight-->Spark Project Scala
5)Tab till Build Tools-->Maven
6)Tab till Next button and press enter
7)Fill all the required details & Tab till finish button and press enter
8)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
9)Navigate to New-->Scala Class and press enter
10)Verify whether screen reader(JAWS) is reading the dialogue name 'Create new scala class'

**Actual Result:**
Jaws not reading the dialogue name 'Create new scala class'

**Expected Result:**
JAWS should read the dialogue name 'Create new scala class'

**User Impact:**
Visually disabled users will find difficult, if the screen reader (JAWS) is not reading the dialogue name

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by t-rufang** [comment]
Note: Screen reader is not reading the dialog name `Create New Scala Class`. This scenario should be supported by IntelliJ.
![image](https://user-images.githubusercontent.com/32627233/65674017-78480e00-e07e-11e9-9576-50d2077befeb.png)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749723
